packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -522,6 +529,12 @@ test.serial('LP withdraws', async t => {
 test.serial('restart contract', async t => {
   const { EV } = t.context.runUtils;
   await null;
+  if (defaultManagerType === 'xs-worker') {
+    t.is(defaultManagerType, 'xs-worker');
```

Comment:
this confused me. A more direct expression of the intent is,
```suggestion
    t.pass();
```